src/commands/app/list/extension-points.js

```diff
@@ -14,7 +14,6 @@ const BaseCommand = require('../../../BaseCommand')
 const aioLogger = require('@adobe/aio-lib-core-logging')('@adobe/aio-cli-plugin-app:list:extension-points', { provider: 'debug' })
 const { Flags } = require('@oclif/core')
 
-const { EXTENSION_POINT_LIST } = require('../../../lib/defaults')
```

Comment:
Can remove from defaults, looks to be unused elsewhere

Reply:
good catch, will update

Reply:
updated with new commit 0f9be2d